rc/base/lint.kak

```diff
@@ -70,6 +80,10 @@ define-command -hidden lint-show %{
         fi
     } }
 
+define-command -hidden lint-show-counters %{
+    echo -markup linting results:{red} %opt{lint_error_count} error(s){yellow} %opt{lint_warning_count} warning(s)
```

Comment:
I think the `Error` and `Prompt faces should be used here, instead of hard-coded colors. The result should be same , with the default theme.

Reply:
Yeah, I think that would make sense.

Reply:
I used explicit `{red}` and `{yellow}` colors to mirror the same hardcoded value of the `█` blocks in the gutter:
```
flags = flags ":" $2 "|{red}█"
flags = flags ":" $2 "|{yellow}█"
```

In default.kak:
```
face global Information black,yellow
face global Error black,red
```
these two faces use yellow and red as background, but `█` needs it as foreground.

Reply:
Right, yeah, thats reasonable.